robottelo/test.py

```diff
@@ -238,11 +238,21 @@ def take_screenshot(self, testmethodname):
         testmethodname = "_".join(testmethodname)
         # Creates dir_structure depending upon the testmethodname.
         directory = testmethodname
-        if not os.path.exists("/tmp/screenshots/" + directory):
-            os.makedirs("/tmp/screenshots/" + directory)
-        self.browser.save_screenshot('/tmp/screenshots/{dir}/'
-                                     'screenshot-{date}.png'
-                                     .format(dir=directory, date=filename))
+        dir_structure = '{screenshots_dir}/{date}/{dir}'.format(
```

Comment:
Was just checking, found unix just handles `//` safely. But agreed os.path,join is the best way to do.